sentry.server.config.ts

```diff
@@ -0,0 +1,5 @@
+import { init } from '@sentry/nextjs';
+
+import { SENTRY_DSN } from '@/next.constants.mjs';
+
+init({ dsn: SENTRY_DSN, tracesSampleRate: 1, debug: false });
```

Comment:
debug: false is the default

```suggestion
init({ dsn: SENTRY_DSN, tracesSampleRate: 1 });
```